### ISSUE TITLE
studio.h file not found

### ISSUE THREAD

**Issue by maedox** [termux/termux-packages#683]
I upgraded the python package and now I can't install bpython for 3.6. 

```
~ $ pip install bpython
...
    creating build/temp.linux-aarch64-3.6
    aarch64-linux-android-clang -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Os -Os -fPIC -I/data/data/com.termux/files/usr/include/python3.6m -c greenlet.c -o build/temp.linux-aarch64-3.6/greenlet.o
    In file included from greenlet.c:5:
    In file included from ./greenlet.h:8:
    /data/data/com.termux/files/usr/include/python3.6m/Python.h:25:10: fatal error: 'stdio.h' file not found
    #include <stdio.h>
             ^
    1 error generated.
    error: command 'aarch64-linux-android-clang' failed with exit status 1
```

I have bpython installed for the previous Python, so it has worked.
python-dev is installed.

```
~ $ find /data/data/com.termux/files/ -name 'stdio.h'
/data/data/com.termux/files/usr/include/libandroid-support/stdio.h
/data/data/com.termux/files/usr/include/stdio.h
```

**Comment by vishalbiswas** [comment]
This is an error in the latest build of clang

**Comment by fornwall** [comment]
It's indeed a problem with clang - I'm rebuilding the clang package with a fix and push it as soon as it's built, I'll write a message here shortly!

**Comment by fornwall** [comment]
@maedox This has now been fixed in version `3.9.1-2` of the clang package, install it with:

    apt update && apt upgrade

**Comment by maedox** [comment]
Great! Works like a charm. Thank you.
